src/index.html

```diff
@@ -9,6 +9,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- Moyo header -->
+    <header class="header">
+      <img
+      src="images/logo.png"
+      alt="header icon"
+      class="header__icon"
+      >
```

Comment:
this also should work as a link to the main page of the site

Reply:
<img width="943" alt="image" src="https://user-images.githubusercontent.com/90685701/167165060-6828204d-5381-43a7-8a0e-9e272c6a9d45.png">